pkg/execution/realtime/realtime.go

```diff
@@ -149,18 +166,27 @@ type ReadWriteSubscription interface {
 // Each message is published to one or more topics.
 type Topic struct {
 	// Kind represents the topic kind, ie. whether this topic is for events or run data.
+	// This allows consumers to stream events and data from runs separately.
 	Kind TopicKind `json:"kind"`
-	// RunID represents the run that this topic represents, if this is a
-	// topic for a run.
-	RunID ulid.ULID `json:"run_id"`
```

Comment:
Can we keep `RunID` in the `Topic`?

`Channel` is the run ID atm but that could change, and keeping the `RunID` here means we also don't need to do `ulid.MustParse()` checks on `Channel` and it can remain _just a string_ which feels better.